homework_13/BubbleSort.java

```diff
@@ -0,0 +1,15 @@
+public class BubbleSort {
+    static int[] bubbleSort(int[] array) {
+        int temp;
+        for (int i = 0; i < array.length - 1; i++) {
+            for (int j = i + 1; j < array.length; j++) {
```

Comment:
This line `for (int j = i + 1; j < array.length; j++)` can be optimized like this: `for (int j = 0; j < array.length - I - 1; j++)`